cookiecutter.json

```
@@ -5,6 +5,7 @@
     "email": "name@alexandra.dk",
     "open_source": "Is this a public open source project? (y/n, default: n)",
     "python_version": "3.11",
+    "_docker-uv-version": "0.6.11",
```

Changing to snake case for consistency:

```suggestion
    "_docker_uv_version": "0.6.11",
```